pkg/logging/logging.go

```diff
@@ -0,0 +1,103 @@
+// package logging is a small wrapper around github.com/k8snetworkplumbingwg/cni-log
+
+package logging
+
+import (
+	cnilog "github.com/k8snetworkplumbingwg/cni-log"
+)
+
+const (
+	labelCNIName     = "cniName"
+	labelContainerID = "containerID"
+	labelNetNS       = "netns"
+	labelIFName      = "ifname"
+	cniName          = "sriov-cni"
+)
+
+var (
+	logLevelDefault = cnilog.ErrorLevel
+	containerID     = ""
+	netNS           = ""
+	ifName          = ""
+)
+
+func init() {
+	SetLogFile("")
+	SetLogLevel("")
+}
+
+// SetLogLevel sets the log level to either verbose, debug, info, warn, error or panic. If an invalid string is
+// provided, it uses error.
+func SetLogLevel(l string) {
+	ll := cnilog.StringToLevel(l)
+	if ll == cnilog.InvalidLevel {
+		ll = logLevelDefault
+	}
+	cnilog.SetLogLevel(ll)
+}
+
+// SetLogFile sets the log file for logging. If the empty string is provided, it uses stderr.
+func SetLogFile(fileName string) {
+	if fileName == "" {
```

Comment:
no needed if you remove it from init and set in main.go only if it is not empty.

Reply:
if you set the default level in init, then you can do:

if ll:= cnilog.StringToLevel(l), ll != cnilog.InvalidLevel {
cnilog.SetLogLevel(ll)
}